compiler/modules/nback/scaffold.bal

```diff
@@ -339,15 +350,15 @@ function addRuntimeFunctionDecl(llvm:Module mod, RuntimeFunction rf) returns llv
     return f;
 }
 
-function addStringDefn(llvm:Context context, llvm:Module mod, int defnIndex, string str) returns llvm:ConstPointerValue {
+function addStringDefn(llvm:Context context, llvm:Module mod, int defnIndex, string str, boolean isDecStr) returns llvm:ConstPointerValue {
```

Comment:
I don't want to make this handle decimal constants also. Deal with decimal constants separately.

Reply:
Fixed